generators/app/package-json.js

```diff
@@ -142,7 +141,7 @@ const packageJSON = (current, context) => {
       'clean': 'shx rm -rf ./dist ./test/dist',
       'postclean': 'shx mkdir -p ./dist ./test/dist',
       'lint': 'vjsstandard',
-      'prepublish': 'not-in-install && npm run build && pkg-ok || in-install',
+      'prepublishOnly': 'npm run build && vjsverify',
```

Comment:
not-in-install was not working correctly for me with `npm@6`, this may be something we have to update our projects with right away.

Reply:
Can `not-in-install` be fixed for npm@6 rather than opening a bunch more PRs?

Reply:
good question let me check